students/roviram/lesson06/caught_speeding.py

```diff
@@ -0,0 +1,44 @@
+#---------------------------------------------------#
+# Dev: Miguel Rovira-Gonzalez
+# Week 6 Homework
+# Script Name: Caught Speeding
+# Script Creation Date: Saturday 5/18/2019
+#---------------------------------------------------#
+"""
+You are driving a little too fast, and a police officer stops you.
+Write code to compute the result, encoded as an int value: 0=no ticket, 1=small ticket, 2=big ticket.
+    If speed is 60 or less, the result is 0.
+    If speed is between 61 and 80 inclusive, the result is 1.
+    If speed is 81 or more, the result is 2.
+    Unless it is your birthday -- on that day, your speed can be 5 higher in all cases.
+
+caught_speeding(60, False) → 0
+caught_speeding(65, False) → 1
+caught_speeding(65, True) → 0
+"""
+
+
+def caught_speeding(speed, is_birthday):
```

Comment:
You can apply this logic a bit neater. One way to think about this is to try and only handle the True or False cases (just one of them). Once you apply those and if you still haven't returned out as true, then return False.

Ex:

if blah and blah: #truth case
return true

if blah and blah2: #second truth case
return true

if blah and blah3: #last truth case
return true

return false #if we haven't returned true by now, we know that we MUST be false